saleor/graphql/webhook/tests/test_webhook.py

```diff
@@ -305,6 +305,24 @@ def test_webhook_delete_by_inactive_app(app_api_client, webhook):
     assert_no_permission(response)
 
 
+@patch("saleor.webhook.models.Webhook.delete")
+def test_webhook_delete_deactivates_before_deletion(
```

Comment:
Wouldn't be good to have a separate test, that will check that ValidationError has been raised?